<launch>

  <include file="$(find iai_tracy_description)/launch/upload.launch"/>

  <node pkg="giskardpy" type="giskard.py" name="giskard" output="screen">
    <param name="config" value="Tracebot_StandAlone"/>
  </node>

  <node pkg="giskardpy" type="interactive_marker.py" name="giskard_interactive_marker" output="screen">
    <rosparam param="enable_self_collision">False</rosparam>
    <rosparam param="interactive_marker_chains">
      - [tracy/table, tracy/l_gripper_tool_frame]
      - [tracy/table, tracy/r_gripper_tool_frame]
    </rosparam>
  </node>

</launch>

